doc/systest_basics.rst

```diff
@@ -0,0 +1,288 @@
+
+Correlation functions
+---------------------
+
+Correlation functions are child classes
+of :class:`stile.sys_tests.BaseCorrelationFunctionSysTest`. You use the
+function :func:`stile.CorrelationFunctionSysTest <stile.sys_tests.CorrelationFunctionSysTest>` to
+create them.  They're all wrappers for
+Mike Jarvis's `TreeCorr <https://github.com/rmjarvis/TreeCorr/>`_ code, so you'll need to have
+that installed to use them.  The predefined types are:
+
+- **GalaxyShear**: tangential and cross shear of galaxies around lenses (point-shear)
+- **BrightStarShear**: tangential and cross shear of galaxies around bright stars (point-shear)
+- **StarXGalaxyDensity**: number density of galaxies around stars (point-point)
+- **StarXGalaxyShear**: shear-shear cross correlation of galaxies and stars (shear-shear)
+- **StarXStarShear**: autocorrelation of the shapes of stars (shear-shear)
+- **StarXStarSize**: autocorrelation of the size residuals for stars, relative to PSF sizes
+  (scalar-scalar)
+- **GalaxyDensityCorrelation**: position autocorrelation of galaxies (point-point)
+- **StarDensityCorrelation**: position autocorrelation of stars (point-point)
+- **Rho1**: rho1 statistics (autocorrelation of residual star shapes)  (shear-shear)
+- **None**: creates a more flexible but less automatic class.  Instead of the usual call signature,
+  which just takes one or more data sets, the first argument to an object created this way must
+  be a TreeCorr correlation function type (such as ``gg`` or ``ng``).  This object also won't
+  have the ``.objects_list`` and ``.required_quantities`` described above.
+
+Calling these objects returns a formatted NumPy array of the correlation functions.  If the order
+of the data sets matters--eg for **GalaxyShear**--any data set that only needs positions should go
+first, and any data set with a scalar value you're using should go last.  For example:
+
+>>> bright_star_shear = stile.CorrelationFunctionSysTest('BrightStarShear')
+>>> results = bright_star_shear(bright_star_data, galaxy_data)
+
+The plotting method returns a :class:`matplotlib.figure.Figure` instance that can be saved
+by :func`.savefig` or further altered if you like.
+
+The estimators are different depending on the type.  Point-point estimates by default use
+the Landy-Szalay estimator:
+
+.. math::
+        \xi = (DD-2DR+RR)/RR
+
+and must include a random catalog.
+
+All shears in the following descriptions are the complex form in the frame aligned with the
+vector between the two points, that is, :math:`g = \gamma_t + i\gamma_x`.
+
+Point-shear estimates are equivalent to average tangential shear, returned as real
+:math:`\langle \gamma_t \rangle` and imaginary :math:`\langle \gamma_x \rangle`.  If random
+catalogs are given, they are used as random lenses, and
+:math:`\langle \gamma_t \rangle-\langle \gamma_{t, {\rm rand}} \rangle` is returned instead.
+
+Shear-shear correlation functions are :math:`\xi_+` and :math:`\xi_-`.  :math:`\xi_+` is,
```

Comment:
Honestly, I understood this because I understood these definitions already, but I'm not sure a newbie would get it.  Perhaps just say "the definitions of xi+ and xi- follow the formalism of BLAH" (where BLAH is one of the many papers that defines this xi+ / xi- formalism) and not try to do this in words?